internal/app/wwctl/container/exec/child/main.go

```diff
@@ -34,18 +36,21 @@ func CobraRunE(cmd *cobra.Command, args []string) (err error) {
 		os.Exit(1)
 	}
 	conf := warewulfconf.Get()
+	if overlayDir == "" {
+		overlayDir = path.Join(conf.Paths.WWChrootdir, "overlays")
```

Comment:
Isn't this the original complaint? My understanding of #1180 was that there shouldn't be anything created in `conf.Paths.WWChrootdir` that isn't a chroot.